dist/index.html

```diff
@@ -11,14 +11,15 @@
 </head>
 <body>
   <header><h1>TRAVEL TRACKER</h1><h1 class ='total-spent'></h1></header>
-  <nav>
+  <nav class='hidden'>
```

Comment:
This makes the login screen correctly tabable though lighthouse acts weird because of the login screen and passes you anyway I think.